test/onnx/onnx_test_common.py

```diff
@@ -395,11 +402,17 @@ def _compare_pytorch_onnx_with_ort(
         ref_model(*ref_input_args, **ref_input_kwargs)
     )
     ort_outputs = run_ort(export_output, onnx_format_args)
+
     if len(ref_outputs) != len(ort_outputs):
         raise AssertionError(
             f"Expected {len(ref_outputs)} outputs, got {len(ort_outputs)}"
         )
     for ref_output, ort_output in zip(ref_outputs, ort_outputs):
+        # NOTE: ONNX Runtime doesn't support tensor of complex64/complex128, so we
+        # convert them to float32/float64.
+        # TODO: Need stft enabled to test complex64/complex128
+        if isinstance(ref_output, torch.Tensor) and torch.is_complex(ref_output):
```

Comment:
You should add a new input/output adaptor step for this. Checkout `RemoveNonTensorInputStep` for example.
Feel free to do it in this or follow-up PR.

Reply:
Done